packages/app/utils/errors.ts

```diff
@@ -15,15 +15,15 @@ export enum ErrorCode {
 
 export interface ErrorCause {
     status: number
-    code: ErrorCode
+    code: ErrorStatusText
 }
 
-export type HttpExceptionStatus = 400 | 401 | 403 | 404 | 405 | 500
+export type ErrorStatusCode = 400 | 401 | 403 | 404 | 405 | 500
```

Comment:
I updated this to `ErrorStatusCode` because it represents [response.status](https://developer.mozilla.org/en-US/docs/Web/API/Response/status) (the value will be once of the HTTP status codes that correspond to an error).